docs/analyzer/languages.md

```diff
@@ -96,7 +97,7 @@ and [Stanza](https://github.com/stanfordnlp/stanza). To set up new models, follo
         from presidio_analyzer.nlp_engine import NlpEngineProvider
 
         # Create NLP engine based on configuration file
-        provider = NlpEngineProvider(conf_file="PATH_TO_YAML")
+        provider = NlpEngineProvider(conf_file="./docs/analyzer/languages-config.yml")
```

Comment:
maybe use environment variable?

Reply:
@balteravishay in this case I will need to set the env var in the pipeline, making it less generic. What do you have in mind?

Reply:
so maybe just a static const var?

Reply:
done